client/server/lib/analytics/index.js

```diff
@@ -42,22 +40,6 @@ function getUserFromRequest( request ) {
 }
 
 const analytics = {
-	statsd: {
-		recordTiming: function ( featureSlug, eventType, duration ) {
```

Comment:
These wrappers weren't really adding any value, so I consolidated everything to `client/lib/analytics`